build-npm-module/index200.js

```diff
@@ -1,5 +1,7 @@
-/**
- * Create a README.md under your case_converter directory.
- *
- * This will be the description and instruction file for your package. It will shou on your module's npm home page.
- */
+/*
+Let's edit your README.md under your case_converter directory.
```

Comment:
For this step we can imagine that the `README.md` file is already created and open. We could break the next step down further by prompting the learner to enter the markdown text themselves.